docs/docs/getting-started/first-steps.md

```diff
@@ -40,181 +40,55 @@ If you don't have a cloud subscription, check out [MiniConstellation](first-step
 2. Fill in your cloud provider specific information.
 
     <tabs groupId="csp">
-    <tabItem value="azure" label="Azure (CLI)">
 
-    You need several resources for the cluster. You can use the following `az` script to create them:
+    <tabItem value="azure" label="Azure">
 
     ```bash
-    RESOURCE_GROUP=constellation # enter name of new resource group for your cluster here
-    LOCATION=westus # enter location of resources here
-    SUBSCRIPTION_ID=$(az account show --query id --out tsv)
-    SERVICE_PRINCIPAL_NAME=constell
-    az group create --name "${RESOURCE_GROUP}" --location "${LOCATION}"
-    az group create --name "${RESOURCE_GROUP}-identity" --location "${LOCATION}"
-    az ad sp create-for-rbac -n "${SERVICE_PRINCIPAL_NAME}" --role Owner --scopes "/subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${RESOURCE_GROUP}" | tee azureServiceAccountKey.json
-    az identity create -g "${RESOURCE_GROUP}-identity" -n "${SERVICE_PRINCIPAL_NAME}"
-    identityID=$(az identity show -n "${SERVICE_PRINCIPAL_NAME}" -g "${RESOURCE_GROUP}-identity" --query principalId --out tsv)
-    az role assignment create --assignee-principal-type ServicePrincipal --assignee-object-id "${identityID}" --role 'Virtual Machine Contributor' --scope "/subscriptions/${SUBSCRIPTION_ID}"
-    az role assignment create --assignee-principal-type ServicePrincipal --assignee-object-id "${identityID}" --role 'Application Insights Component Contributor' --scope "/subscriptions/${SUBSCRIPTION_ID}"
-    echo "subscription: ${SUBSCRIPTION_ID}
-    tenant: $(az account show --query tenantId -o tsv)
-    location: ${LOCATION}
-    resourceGroup: ${RESOURCE_GROUP}
-    userAssignedIdentity: $(az identity show -n "${SERVICE_PRINCIPAL_NAME}" -g "${RESOURCE_GROUP}-identity" --query id --out tsv)
-    appClientID: $(jq -r '.appId' azureServiceAccountKey.json)
-    clientSecretValue: $(jq -r '.password' azureServiceAccountKey.json)"
+    constellation iam create azure --region=westus --resourceGroup=constellTest --servicePrincipal=spTest
     ```
 
-    Fill the values produced by the script into your configuration file.
+    :::info
+    This command creates IAM configuration on the Azure region `westus` creating a new resource group `constellTest` and a new service principal `spTest`.
+    :::
 
     :::tip
-
-    Alternatively, you can leave `clientSecretValue` empty and provide the secret via the `CONSTELL_AZURE_CLIENT_SECRET_VALUE` environment variable.
-
+    Since `clientSecretValue` is a sensitive value, you can leave it empty in the configuration file and pass it via an environment variable instead. To this end, create the environment variable `CONSTELL_AZURE_CLIENT_SECRET_VALUE` and set it to the secret value.
```

Comment:
You mean to duplicate "Fill in your cloud provider specific information" into all of the tabs?

Reply:
Yes. Or alternatively, maybe remove the tip and have it only in the workflow? Or something else?

Reply:
I think removing it here is the way to go. I was unsure because it was a tip with security-impact, but i assume people who take enough care to not put the bare secret into the config file will also read the workflow page